### ISSUE TITLE
IndexTools broken on python 3.7

### ISSUE THREAD

**Issue by jdidion** [dnanexus/IndexTools#32]
There appears to be a difference between 3.6 and 3.7 with the attributes available on types:

```
root@39d3db5b72a7:/# which indextools 
/usr/local/bin/indextools
root@39d3db5b72a7:/# indextools
Traceback (most recent call last):
  File "/usr/local/bin/indextools", line 6, in <module>
    from indextools.console import indextools
  File "/usr/local/lib/python3.7/site-packages/indextools/console/__init__.py", line 37, in <module>
    ac.conversion(decorated=parse_region)
  File "/usr/local/lib/python3.7/site-packages/autoclick/types/__init__.py", line 68, in conversion
    return decorator(decorated)
  File "/usr/local/lib/python3.7/site-packages/autoclick/types/__init__.py", line 63, in decorator
    click_type = ParamTypeAdapter(_dest_type.__name__, target)
  File "/usr/local/lib/python3.7/typing.py", line 702, in __getattr__
    raise AttributeError(attr)
AttributeError: __name__
```

**Comment by jdidion** [comment]
This is an issue with autoclick.

**Comment by jdidion** [comment]
Fixed with upgrade to autoclick 0.6.1
